### DISCUSSION TITLE
How to use the production build when hosting on GitHub pages

### DISCUSSION THREAD

**Discussion by george-gca** [slidevjs/slidev#1410] [Q&A] [answered]
I created a slide and hosted it in [GitHub pages](https://george-gca.github.io/bracis_2023_srcaps/). If you look at the console in the browser dev tools, it displays: `You are running a development build of Vue. Make sure to use the production build (*.prod.js) when deploying for production.` How do I fix this?

**Comment by kermanx** [comment]
Currently, Slidev always uses the DEV version of Vue for a very long time and I don't know why😭

It seems that using the DEV version doesn't cause problems?

**Comment by kermanx** [comment, marked as answer]
resolved via #1466
